srcalc/src/main/java/gov/va/med/srcalc/util/DisplayNameConditions.java

```diff
@@ -0,0 +1,31 @@
+package gov.va.med.srcalc.util;
+
+import java.util.regex.Pattern;
+
+public class DisplayNameConditions
```

Comment:
Thanks for doing this, but please add a class Javadoc. Also you need to update various Javadocs to properly link to this class. (Try `gradle javadoc`.)